diff -ru xsd-2.3.1-i686-windows/libxsd/xsd/cxx/tree/buffer.txx xsd/libxsd/xsd/cxx/tree/buffer.txx
--- xsd-2.3.1-i686-windows/libxsd/xsd/cxx/tree/buffer.txx	Fri Jan 19 17:28:00 2007
+++ xsd/libxsd/xsd/cxx/tree/buffer.txx	Tue Oct 23 19:53:39 2007
@@ -94,7 +94,7 @@
       {
         if (this != &other)
         {
-          capacity_ (other.capacity_, false);
+          capacity (other.capacity_, false);
           size_ = other.size_;
 
           if (size_)
diff -ru xsd-2.3.1-i686-windows/libxsd/xsd/cxx/tree/containers.hxx xsd/libxsd/xsd/cxx/tree/containers.hxx
--- xsd-2.3.1-i686-windows/libxsd/xsd/cxx/tree/containers.hxx	Fri Jan 19 17:28:00 2007
+++ xsd/libxsd/xsd/cxx/tree/containers.hxx	Wed Oct 24 05:47:22 2007
@@ -928,13 +928,7 @@
         {
         }
 
-        explicit
-        sequence (size_type n)
-            : sequence_common (n, X ())
-        {
-        }
-
-        sequence (size_type n, const X& x)
+        sequence (size_type n, const X& x = X())
             : sequence_common (n, x)
         {
         }
@@ -970,14 +964,9 @@
         }
 
       public:
-        void
-        resize (size_type n)
-        {
-          sequence_common::resize (n, X ());
-        }
 
         void
-        resize (size_type n, const X& x)
+        resize (size_type n, const X& x = X())
         {
           sequence_common::resize (n, x);
         }
diff -ru xsd-2.3.1-i686-windows/libxsd/xsd/cxx/tree/list.hxx xsd/libxsd/xsd/cxx/tree/list.hxx
--- xsd-2.3.1-i686-windows/libxsd/xsd/cxx/tree/list.hxx	Fri Jan 19 17:28:00 2007
+++ xsd/libxsd/xsd/cxx/tree/list.hxx	Wed Oct 24 05:43:19 2007
@@ -40,13 +40,7 @@
         {
         }
 
-        explicit
-        list (typename sequence<X>::size_type n)
-            : sequence<X> (n, X ())
-        {
-        }
-
-        list (typename sequence<X>::size_type n, const X& x)
+        list (typename sequence<X>::size_type n, const X& x = X())
             : sequence<X> (n, x)
         {
         }
